src/lib/components/search-attribute-filter/list-filter.svelte

```diff
@@ -3,51 +3,79 @@
 
   import Button from '$lib/holocene/button.svelte';
   import ChipInput from '$lib/holocene/input/chip-input.svelte';
+  import Input from '$lib/holocene/input/input.svelte';
   import { translate } from '$lib/i18n/translate';
+  import { isInConditional } from '$lib/utilities/is';
+  import { formatListFilterValue } from '$lib/utilities/query/search-attribute-filter';
 
+  import ConditionalMenu from './conditional-menu.svelte';
   import { FILTER_CONTEXT, type FilterContext } from './index.svelte';
 
   const { filter, handleSubmit } = getContext<FilterContext>(FILTER_CONTEXT);
 
-  $: ({ value } = $filter);
-  $: list =
-    value.length > 0
-      ? value
-          .slice(1, -1)
-          .split(', ')
-          .map((v) => v.slice(1, -1))
-      : [];
+  $: ({ value, conditional } = $filter);
+  $: _value = value;
+  $: chips = formatListFilterValue(_value);
+  $: options = [
+    { value: 'in', label: 'In' },
+    { value: '=', label: translate('common.equal-to') },
```

Comment:
> What are the uses of the conditionals that aren't "In"? Like is "Equal to" checking the exact list? "Dog, Cat, Bird" Equal to "Dog, Cat, Bird"?

@Alex-Tideman say a workflow has a `CustomKeywordListField` with the values `keywordA` and `keywordB`. Checking for `CustomKeywordListField = keywordA` or `CustomKeywordListField = keywordB` would return the workflow, checking for`CustomKeywordListField = keywordC` would not. Checking for `CustomKeywordListField IN ("keywordA", "keywordB", "keywordC")` would also return the workflow even though `keywordC` is not a defined value for the workflow. 

Not sure on usefulness of having both options (`CustomKeywordListField = keywordA`  and `CustomKeywordListField In ("keywordA")` are essentially the same), but I figure if it's possible to have in the search input we should support it in the filter search.